src/convert/resourcesAndReadingOrder.ts

```diff
@@ -98,6 +102,11 @@ const manifestToLink =
       id: manifest.ID,
     };
 
+    const enc = getLinkEncryption(epub, relativePath);
```

Comment:
Does it make sense to do a/the check for the `EncryptionDoc` here? If it's `undefined` then we can skip this section. I know practically that this makes no difference as the `getLinkEncryption`method handles this, but it feels easier to follow to me.

Reply:
I figured we should keep the check for `EncryptionDoc` in the `getLinkEncryption` because otherwise we would have to put two checks anytime we want to use `getLinkEncryption`: 1. check for presence of `EncryptionDoc`, 2. check that `getLinkEncryption` returned something (there was indeed an entry found in the `EncryptionDoc`). I am thinking about `getLinkEncryption` as "get it if it exists" if that makes sense?